test/karma.conf.js

```diff
@@ -187,6 +187,8 @@ module.exports = function setKarmaConfig(config) {
           path: false,
           // needed by enzyme > cheerio
           stream: false,
+          // required by enzyme > cheerio > parse5
+          util: require.resolve('util/'),
```

Comment:
I don't know which package caused this, but after upgrading `next` I had to add this polyfill:
https://app.circleci.com/pipelines/github/mui/material-ui/71594/workflows/3f387e20-f92a-481c-bdec-d22fcf0bd2f3/jobs/380549?invite=true#step-112-873

Reply:
Have you tried `util: false`? Locally it didn't fail here (I ran `git clean -fdx && yarn install` before). It's weird because next is not used to run the tests.

Reply:
> Have you tried util: false? 

I tried it before - it didn't work. I just tried it now again - it worked. But then I uninstalled `util` package - and it fails again :D
So basically if the package is installed - it doesn't matter what's in `resolve`

Reply:
I found the reason it worked before. `utils` was a dependency of next [v12.0.7](https://github.com/vercel/next.js/blob/ebf509539c0f4f519923c7780e269872059c384e/packages/next/package.json#L116). In [v12.1.5](https://github.com/vercel/next.js/blob/38d17bca0c34ebcc3055dbdfc42bfff26f88f62e/packages/next/package.json#L264) it became a devDependency, so not installed automatically. We're not adding a new dependency here, only making it required.

Reply:
Thanks for looking into this!
So we can keep it in the current state, right?

Reply:
I would keep only because of the comment, but it's not necessary.